### PR TITLE
mag doc updates!!!

### PR DESCRIPTION
**Pull request by nivasan1**

### CODE REVIEW

File: docs/about/1-faq.md [hunk 1]
```diff
@@ -6,61 +6,40 @@ sidebar_position: 1
...
 
 ### 🤔 Does Skip have a token?
 
-No, Skip doesn’t have a token.
+No.
 
 ### 🤔 How many chains is Skip currently live on?
 
-Skip is currently live on 20+ chains across it's Block SDK, API, ProtoRev and Skipper products.
+Skip is currently live on 30+ chains across Slinky, API, and Block SDK, and other deployments, including Osmosis, dYdX, Berachain, Neutron, and Celestia... and more!
```
Comment (Eric-Warehime) [inline docs/about/1-faq.md:29]:
```suggestion
Skip is currently live on 30+ chains across Slinky, the API, the Block SDK, and other deployments, including Osmosis, dYdX, Berachain, Neutron, Celestia... and more!
```

File: docs/about/0-intro.md [hunk 2]
```diff
@@ -7,27 +7,27 @@ sidebar_position: 0
...
 
-# What does Skip offer?
+# ⛓️ Skip On-Chain Products
 
-### 🧱 For Chains
+### 🔮 Slinky
 
-Skip builds Slinky(/docs/chains/slinky/0-overview.md), an enshrined oracle that integrates with Cosmos chains using ABCI++ to provide a simple, secure, and reliable way to post data on-chain.
-If you're interested in integrating Slinky into your chain please [contact us](https://skip.money/contact).
+Skip builds [Slinky](/slinky/overview), an enshrined oracle for the highest-performance DeFi that leverages your chain's native validator set to provide the fastest, most secure on-chain updates, guaranteed every block.
```
Comment (Eric-Warehime) [inline docs/about/0-intro.md:19]:
```suggestion
Skip builds [Slinky](/docs/slinky/overview), an enshrined oracle for the highest-performance DeFi that leverages your chain's native validator set to provide the fastest, most secure on-chain updates, guaranteed every block.
```

File: docs/about/0-intro.md [hunk 3]
```diff
@@ -7,27 +7,27 @@ sidebar_position: 0
...
 
-We also built an open-source [Block SDK](/docs/chains/blocksdk/1-overview.md) that allows for full control over how blocks are built, including a MEV recapture mechanism, a sovereign Oracle, and segmentation to tailor blocks to your app-chain use case.
+If you're interested in integrating Slinky, please <b>[contact us](https://skip.money/contact).</b>
 
-Used by the majority of major Cosmos SDK chains, we recommend the [Block SDK](/docs/chains/blocksdk/0-integrate-the-sdk.md) for any chains running Cosmos SDK version 0.47 or higher.
-Integration takes < 20 mins, and please [contact us](https://skip.money/contact) if you need assistance.
+### 🧱 Block SDK
 
-### 🥷 For Searchers
+We also build the [Block SDK](/blocksdk/overview), an open-source module that gives full control over how blocks are built, including a prebuilt MEV recapture mechanism, a EIP-1559 fee market, and segmentation to tailor blocks to your app-chain use case.
```
Comment (Eric-Warehime) [inline docs/about/0-intro.md:25]:
```suggestion
We also build the [Block SDK](/docs/blocksdk/1-overview), an open-source module that gives full control over how blocks are built, including a prebuilt MEV recapture mechanism, a EIP-1559 fee market, and segmentation to tailor blocks to your app-chain use case.
```

File: docs/about/0-intro.md [hunk 4]
```diff
@@ -7,27 +7,27 @@ sidebar_position: 0
...
 
-On both chains that use Skip Block SDK, searchers can participate in capturing MEV and bidding in the top-of-block auction for atomic execution. Check out our searcher docs for Skip [Block SDK](/docs/0-searcher-docs.md).
+If you're interested in integrating, check out our <b>[integration docs](/blocksdk/0-integrate-the-sdk)!</b>
```
Comment (Eric-Warehime) [inline docs/about/0-intro.md:27]:
```suggestion
If you're interested in integrating, check out our <b>[integration docs](/docs/blocksdk/0-integrate-the-sdk)!</b>
```

**Review by Eric-Warehime — COMMENTED** [review]
There's a bunch of broken links which are causing the build to fail, otherwise looks good :D

**Comment by MaghnusM** [comment]
Okay should be fixed @Eric-Warehime

**Review by davidterpay — APPROVED** [review]

**Review by Eric-Warehime — APPROVED** [review]
Looks good--concerned about breaking old links by moving things. Not sure if we can setup some redirects where it makes sense--for blocksdk stuff mainly I guess.

**Pull request merged**